### DISCUSSION TITLE
Showing a "AR not supported" prompt/button when ar is not available

### DISCUSSION THREAD

**Discussion by 0xhrsh** [google/model-viewer#4157] [Q&A] [answered]
Is there an option to show a prompt/button if the "AR-Button" is not visible?

**Comment by 0xhrsh** [comment, marked as answer]
So I am opening modelviewer on firefox (ios) and the "ar-button" is not visible because ar is not supported there. In such a case I want to display to the users a prompt to change the device or browser. Is that currently possible?

I tried looking at the CSS and properties of the button, there wasn't anything different in them, else I would have checked for that difference and displayed the prompt myself.

**Reply by elalish** [reply]
I believe you're looking for this: https://modelviewer.dev/docs/index.html#entrydocs-augmentedreality-properties-canActivateAR

**Reply by 0xhrsh** [reply]
I tried this, but this is pretty inaccurate. 

I ended up making a list of supported browsers and checking if the requesting browser is in that list.

**Reply by 0xhrsh** [reply]
thanks for following up!
